server/server.js

```diff
@@ -56,27 +56,75 @@ app.middleware('parse', bodyParser.urlencoded({
   extended: true
 }));
 
-// The access token is only available after boot
-app.middleware('auth', loopback.token({
-  model: app.models.accessToken,
-  currentUserLiteral: 'me'
-}));
 
-app.use(loopback.token());
+
+// Establish current user from access token
+app.use(loopback.context());
+app.use(function setCurrentUser(req, res, next) {
```

Comment:
This injects the user into the current context based on the access token, if present.